zipkin-storage/cassandra-v1/src/test/java/zipkin2/storage/cassandra/v1/InternalForTests.java

```diff
@@ -38,13 +38,14 @@ static void writeDependencyLinks(
     storage.session().execute(statement);
   }
 
-  static String keyspace(TestName testName) {
-    String result = testName.getMethodName().toLowerCase();
-    return result.length() <= 48 ? result : result.substring(result.length() - 48);
-  }
+  static Set<String> USED_KEYSPACES = Collections.synchronizedSet(new HashSet<>());
 
-  static void dropKeyspace(Session session, String keyspace) {
-    session.execute("DROP KEYSPACE IF EXISTS " + keyspace);
-    assertThat(session.getCluster().getMetadata().getKeyspace(keyspace)).isNull();
+  static String randomKeyspace() {
+    String result;
```

Comment:
it would be a little more user friendly for test name based keyspaces. sometimes I run without docker and debugging is a bit more ok that way. logs in Cassandra etc are slightly more friendly with something in the name that has some meaning